.claude-plugin/commands/init.md

```
@@ -14,20 +14,20 @@ For each tool (ba, wm, sg), check if binary exists:
-   ```bash
-   brew install cloud-atlas-ai/ba/ba
-   brew install cloud-atlas-ai/wm/wm
-   brew install cloud-atlas-ai/superego/superego
-   ```
-
-   If **Cargo** available:
+   If **Cargo** available (preferred):
    ```bash
    cargo install ba
    cargo install working-memory  # (published as 'working-memory', provides 'wm' binary)
    cargo install superego
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)